datasets/nasa_wildfire/pipelines/dataset.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  name: nasa_wildfire
+  friendly_name: nasa_wildfire
+  description: NASA Wildfire dataset
+  dataset_sources: ~
+  terms_of_use: ~
+
+resources:
+  - type: bigquery_dataset
+    dataset_id: nasa_wildfire
+    description: Past Week dataset
+  - type: storage_bucket
+    name: nasa-wildfire
+    uniform_bucket_level_access: True
+    location: US
```

Comment:
Remove the storage bucket section (lines 26-29) as they are not used, then rerun generate_terraform to recreate the tf files.

Reply:
@nlarge-google done